masterlist.yaml

```
@@ -6367,10 +6367,13 @@ plugins:
 
   - name: 'SecretsoftheAyleidsClosure.esp'
     url: [ 'https://www.nexusmods.com/oblivion/mods/22918/' ]
+    tag:
+     - Actors.AIPackages
+     - Stats
```

Indentation for the tags is off by a space.

```suggestion
      - Actors.AIPackages
      - Stats
```